control-plane/pkg/apis/messaging/v1beta1/kafka_channel_types_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1beta1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	eventingduck "knative.dev/eventing/pkg/apis/duck/v1"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+func TestKafkaChannel_GetGroupVersionKind(t *testing.T) {
```

Comment:
```suggestion
func TestKafkaChannelGetGroupVersionKind(t *testing.T) {
```
I think we should remove the underscore. Why now and here? looks odd to me

Reply:
Don't know. Copy & paste from knative-sandbox/eventing-kafka